.github/workflows/repo-settings-sync.yml

```
@@ -0,0 +1,31 @@
+---
+name: Repo Setup
+
+on:
+  push:
+    branches:
+      - master
+  schedule:
+    - cron: 0 0 * * *
```

Let's de-sync this off of daily, just to be nice to GitHub's services.

```suggestion
    - cron: 14 12 * * *
```